design/resources.go

```diff
@@ -355,3 +355,25 @@ var _ = Resource("trackerquery", func() {
 	})
 
 })
+
+var _ = Resource("search", func() {
+	BasePath("/search")
```

Comment:
yes @aslakknutsen , type will be another query param (not mandatory).
In current case if not provided we will consider type=workitem.

This is helpful for searching user while assigning tasks to users as well.